MapboxSearch/offline/src/androidTest/java/com/mapbox/search/offline/OfflineSearchEngineIntegrationTest.kt

```diff
@@ -558,17 +557,41 @@ internal class OfflineSearchEngineIntegrationTest {
         fun compareWithApproximateLocations(expected: BaseRawSearchResult, serverResult: BaseRawSearchResult): Boolean {
             if (expected == serverResult) return true
 
-            val fixedResult = expected.copy(
-                id = serverResult.id,
-                userRecordPriority = serverResult.userRecordPriority,
-                center = serverResult.center,
-                distanceMeters = serverResult.distanceMeters
-            )
-            return fixedResult == serverResult &&
-                    expected.center.approximatelyEquals(serverResult.center) &&
-                    compareDistanceMeters(expected.distanceMeters, serverResult.distanceMeters)
+            if (assertSearchResultEquals(expected, serverResult))
+                return true
+
+            Log.d(LOG_TAG, "compareWithApproximateLocations: expected = $expected, serverResult = $serverResult")
+            return false
         }
 
+        fun assertSearchResultEquals(expected: BaseRawSearchResult, actual: BaseRawSearchResult): Boolean {
```

Comment:
I think you can simplify this function a bit by comparing public fields of OfflineSearchResult, instead of BaseRawSearchResult. By doing this you won't have to carry about comparing fileds like `brand` or `userRecordPriority` that are not valid for offline search results.

Reply:
Done